### ISSUE TITLE
Create tool to aggregate V3 package issues found by the V3 monitoring job into a report

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#4046]
Currently, because we have so many packages with issues (most of them being very minor), it's hard to parse through the blobs and aggregate the different issues by hand. We should write a tool that will automatically aggregate the issues in a way that is more human readable.

**Comment by skofman1** [comment]
We should fix V3 issues as they come up. On of the prerequisites of running a monitoring job is that it won't be noisy, so there should be nothing to report. 
Closing this issue for now. We can reopen in the future if such a report is required.
